.rustfmt.toml

```diff
@@ -1 +1,2 @@
 edition = "2018"
+max_width = 120
```

Comment:
Yay!

Reply:
I changed that because rust-analyzer wasn't respecting the limit in rustfmt.toml (we have two of these, need to figure out why).